thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/pojo/FunctionAutoTuneConfigBean.java

```diff
@@ -0,0 +1,131 @@
+package com.linkedin.thirdeye.datalayer.pojo;
```

Comment:
can you request @npawar @puneetjaiswal  to review the database design and coding. Please share a wiki link for the design.